src/boss/lambdastore/store.go

```diff
@@ -11,11 +11,15 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/open-lambda/open-lambda/ol/boss/cloudvm"
+	"github.com/open-lambda/open-lambda/ol/boss/event"
 	"github.com/open-lambda/open-lambda/ol/common"
 )
 
 type LambdaStore struct {
-	StorePath string
+	StorePath    string
```

Comment:
Let's add some comments explaining what is at StorePath and trashPath.  The latter especially should have some explaining (similar to your lower comment on the PR, but as a comment here).